oauth2_provider/urls.py

```diff
@@ -30,5 +30,15 @@
     ),
 ]
 
+oidc_urlpatterns = [
+    re_path(
+        r"^\.well-known/openid-configuration/$",
+        views.ConnectDiscoveryInfoView.as_view(),
+        name="oidc-connect-discovery-info",
+    ),
+    re_path(r"^jwks/$", views.JwksInfoView.as_view(), name="jwks-info"),
```

Comment:
I can't find an RFC where the `jwks` path is defined, and its not defined at [IANA](https://www.iana.org/assignments/well-known-uris/well-known-uris.xhtml#well-known-uris-1). 

All well behaved OIDC clients should discover it from `jwks_uri` from `.well-known/openid-configuration` of course, but it seems *convention* has it `.well-known/jwks.json` ([Auth0](https://auth0.com/docs/tokens/json-web-tokens/json-web-key-sets), [AWS](https://aws.amazon.com/premiumsupport/knowledge-center/decode-verify-cognito-json-token/), [Akamai](https://blogs.akamai.com/2019/10/verify-jwt-with-json-web-key-set-jwks-in-api-gateway.html) etc). Okta *don't*, so it's clearly not a huge deal.

Should I update to `.well-known/jwks.json`?